lib/bibliothecary/parsers/go.rb

```diff
@@ -7,8 +7,12 @@ class Go
       include Bibliothecary::Analyser
 
       GPM_REGEXP = /^(.+)\s+(.+)$/
-      GOMOD_REGEX = /^(require\s+)?(.+)\s+(.+)$/
-      GOMOD_IGNORABLE_REGEX = /^(\/\/|module\s|go\s|exclude\s|replace\s|require\s+\(|\))/m
+      GOMOD_REPLACEMENT_SEPARATOR_REGEX = / => /
```

Comment:
I was not sure what I was looking at initially. WDYT about using the `%r` syntax here to make it more clear this is a regular expression rather than...two divisions with an arrow operator?

Reply:
I figured it'd be clear since the var is suffixed with `_REGEX`, forward-slashes are the ubiquitous delimiter for regexes and it's listed above/below the other regexes, but I'm not opposed to using a different syntax here

Reply:
actually, I'm hesitant to start mixing syntaxes in bibliothecary, so I'll just replace the whitespaces with `\s` to make it more obvious